stubs/gdb/@tests/stubtest_allowlist.txt

```diff
@@ -0,0 +1,113 @@
+gdb.BP_NONE
```

Comment:
I assume that these should be included in the stubs in the future? In that case, could you add a TODO comment at the start? Otherwise, please add a comment explaining why these entries are necessary.